kinetic/distribution.yaml

```diff
@@ -16673,6 +16673,21 @@ repositories:
       url: https://github.com/uos/uos_tools.git
       version: kinetic
     status: maintained
+  ur_client_library:
+    doc:
+      type: git
+      url: https://github.com/UniversalRobots/Universal_Robots_Client_Library.git
+      version: boost
```

Comment:
This is probably fine if it is intentional, but the branch name raised a flag for me so I wanted to point it out in case it was an error. I'll merge anyways, as these could always be updated if need be.